logstash-offline/roles/logstash-ansible/tasks/install-logstash.yaml

```diff
@@ -45,13 +43,10 @@
   template:
     src: "{{ item.src }}"
     dest: "{{ item.dest }}"
+    mode: "{{ item.mode }}"
   with_items:
-    - { src: 99-output.conf.j2, dest: /etc/logstash/conf.d/99-output.conf}
-    - { src: 02-SampleFilter.conf.j2, dest: /etc/logstash/conf.d/02-SampleFilter.conf 
-    - { src: logstash.service.j2, dest: /etc/systemd/system/logstash.service}
-    - { src: startup.options.j2, dest: /etc/logstash/config/startup.options}
-
-- name: Give correct permissions to logstash dir
-  become: yes
-  command: chmod -R 0777 /etc/logstash
+    - { src: 99-output.conf.j2, dest: /etc/logstash/conf.d/99-output.conf, mode: "0700" }
+    - { src: 02-SampleFilter.conf.j2, dest: /etc/logstash/conf.d/02-SampleFilter.conf, mode: "0700" }
+    - { src: logstash.service.j2, dest: /etc/systemd/system/logstash.service, mode: "0700" }
+    - { src: startup.options.j2, dest: /etc/logstash/config/startup.options, mode: "0700" }
```

Comment:
This permission should 0755 since they do not need to be hard restricted to user access only.